### ISSUE TITLE
Bean identification fails for optional (minOccurs=0) multiline records with a record identifying field

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#87]
```
See https://groups.google.com/forum/#!topic/beanio/wp06yxF9lds.

```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 8 Jul 2013 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
Fixed for 2.0.7, although there is a scenario that currently isn't supported 
because the records are not bound to bean objects:

<group name="multirow" class="beanio.Multirow" minOccurs="0" maxOccurs="1" 
order="3">
    <record name="multirowDataRow1" minOccurs="1" maxOccurs="1">
        <field name="rcrdType" rid="true" length="5" literal="MULTI"/>
        <field name="value1" length="unbounded"/>
    </record>
    <record name="multirowDataRow2" minOccurs="0" maxOccurs="1">
        <field name="filler" rid="true" literal=" " ignore="true" />
        <field name="value2" length="unbounded"/>
    </record>
</group>

See thread https://groups.google.com/forum/#!topic/beanio/wp06yxF9lds for 
details.
```

Original comment by `kevin.s...@gmail.com` on 14 Jul 2013 at 5:25

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 7 Sep 2013 at 6:19
- Changed state: **Fixed**
